### ISSUE TITLE
Red Alert lol

### ISSUE THREAD

**Issue by cmvolden** [cmvolden/volden-design-website#1]
Hi @thomasjbradley ! I'm not sure what happened, but I tried pushing a commit and got some weird notification in my README file, so I went in and it asked to choose if it was theirs or ours? So I said ours. Pushed the commit, and then half of my HTML/CSS was no longer working. None of it has disappeared, but it's just not working where it should be. So I took out all of my HTML and CSS, pushed it, then re-added it and pushed it but no change. I then reverted the commit where the original problem occurred. I'm going to stop messing with it now, do you think I should just start a new repository and re-link everything to my Hover/Cloudflare? Or is this something you've encountered before? No rush. Thanks! :D

**Comment by thomasjbradley** [comment]
Send me a screenshot of what it looks like on your computer so I can try to debug.

**Comment by cmvolden** [comment]
Thanks! Okay so here is what it's looking like now, which is different than last night somehow. It's supposed to all be full screen but is all condensed to the left side for some reason, like how it would look on a tablet-sized screen. 
<img width="1792" alt="Screen Shot 2020-10-07 at 10 47 52 AM" src="https://user-images.githubusercontent.com/43150501/95347255-b29bdc00-088a-11eb-9874-25ddfdf85266.png">

**Comment by thomasjbradley** [comment]
Does that match what it looks like on `localhost`?

**Comment by cmvolden** [comment]
Like when you open in browser from Atom? Actually yes..... but it was showing accurately previously.

**Comment by thomasjbradley** [comment]
Okay. Well consistency is good. That’s the first step.
Now we just need to figure out how to fix it.

**Comment by thomasjbradley** [comment]
So, I think, it must be the `m-1-2` sizes on these lines:

- https://github.com/cmvolden/volden-design-website/blob/master/index.html#L41
- https://github.com/cmvolden/volden-design-website/blob/master/index.html#L46

**Comment by cmvolden** [comment]
I just don't understand how that would be affecting it :( It was working before, and its also affecting my nav and footer which should not be happening.

**Comment by cmvolden** [comment]
Also Github desktop keeps giving me the option to Pull origin instead of push...

**Comment by cmvolden** [comment]
Okay I transferred all of my html and css into a new repo just to see if it would work and it did, so I redirected my domain and its working. I don't know what I did back there but it really messed things up. I am now getting a notification when I try to push (see attachment) and I don't know if that's normal? 
<img width="695" alt="Screen Shot 2020-10-07 at 9 54 15 PM" src="https://user-images.githubusercontent.com/43150501/95406445-6aaba200-08e8-11eb-8531-ee4183aa6bb3.png">
